src/frontends/paddle/README.md

```diff
@@ -0,0 +1,32 @@
+# OpenVINO™ Padddle Frontend
+
+OpenVINO Paddle Frontend is one of the OpenVINO Frontend libaries that is dedicated for Baidu PaddlePaddle™ framework. 
+The component is responsible for:
+ * Paddle Reader - component reads PaddlePaddle protobuf model, and parses it to frontend InputModel [paddle frontend architecture](./docs/paddle_frontend_architecture.md).
+ * Paddle Converter - component decodes the PaddlePaddle model and operators, and map them sematically to OpenVINO opset [operator mapping flow](./docs/operation_mapping_flow.md).
+
+The diagram below shows the positions of Paddle Frontend and the main components.![](./docs/img/PaddleFrontendPositioning.PNG)
```

Comment:
Looks like you didn't add the png file. Anyway, can we use `Mermaid` instead of images for diagrams?
Example: https://github.com/openvinotoolkit/openvino/blob/master/src/frontends/ir/docs/architecture.md

Reply:
Uploaded the png files. Actually I have created the files a long time ago, so they could be directly used. Can they be used here?

Reply:
@ceciliapeng2011 I recommend to use mermaid because:

- We have one style for all diagrams in the developer documentation
- In case of any changes it is easier to support (you can completely change the diagram without graphic editor)
- We don't spend a place in git lfs storage

Reply:
For example [this picture](https://github.com/ceciliapeng2011/openvino/blob/cecilia/paddle/doc/src/frontends/paddle/docs/img/PaddleFrontendPositioning.PNG) is out-of date, because:

- We don't have readers and readers API
- This picture shows legacy API (CNNNetwork, nGraph, InferenceEngine)

For [this picture](https://github.com/ceciliapeng2011/openvino/blob/cecilia/paddle/doc/src/frontends/paddle/docs/img/PaddleFrontendInternal.PNG) we don't have IENetworkReader object

Reply:
updated with mermaid